NitroxModel/Helper/NitroxUser.cs

```diff
@@ -93,14 +95,15 @@ public static string GamePath
                     return gamePath;
                 }
 
-                List<string> errors = new();
-                string path = GameInstallationFinder.Instance.FindGame(errors);
-                if (!string.IsNullOrWhiteSpace(path) && Directory.Exists(path))
+                List<GameFinderResult> finderResults = GameInstallationFinder.Instance.FindGame(GameInfo.Subnautica).TakeUntilInclusive(r => r is { IsOk: false }).ToList();
```

Comment:
My only thought as we have now hardcoded as such in the GameInfo.Subnautica how we want this to search in future when we potentially have BZ if this is the default get and if it's worth addressing this now

Reply:
As stated on the PR it's a first PR, others will come after. Previously everything was hardcoded for subnautica inside the finders, that's the aim of using GamInfo